README.md

```diff
@@ -1 +1,40 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello-world
+
+> Description the project.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
```

Comment:
- [ ] Make sure all links in your documentation are working perfectly. The **live demo** link is broken, and as your project is not deployed yet, I suggest you remove this section or deploy the project and add the working link. This will make your documentation more professional.

Reply:
I removed the live demo section from the documentation